source/adapters/level_zero/command_buffer.cpp

```diff
@@ -986,3 +986,93 @@ UR_APIEXPORT ur_result_t UR_APICALL urCommandBufferEnqueueExp(
 
   return UR_RESULT_SUCCESS;
 }
+
+UR_APIEXPORT ur_result_t UR_APICALL urEventGetSyncPointProfilingInfoExp(
+    ur_event_handle_t Event, ur_exp_command_buffer_sync_point_t SyncPoint,
```

Comment:
Variable names should match the UR spec, i.e `hEvent` rather than `Event`, same for other adapters

Reply:
`hEvent` has been updated but the others need to be as well so that they start with a lower case letter.